packages/eslint-plugin-foreman/bin/tfm-lint.js

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env node
+/* eslint-disable no-console */
+
+const path = require('path');
+
+const cwd = process.cwd();
+const chalk = require('chalk');
+const rules = require('../lib/index.js');
+const { CLIEngine } = require('eslint');
+
+let cli = null;
```

Comment:
Would it make sense to implement the `TfmLintCli` as a class?
```js
const linter = new TfmLintCli(files);
linter.lint();
```

Anyway, IMO some of the code here can move to the `lib` folder.